.gitignore

```diff
@@ -25,7 +25,7 @@
 .LSOverride
 
 # Icon must end with two \r
-Icon
+Icon
```

Comment:
This shouldn't be changed manually. Everything after the header is curated from external sources (gitignore.io) so any additions should be in the beginning

Reply:
ok, I didn't know it... I'll revert it to the previous version

Reply:
This is unfortunately still not the original version. The original line is really `Icon\r\r\n`

Reply:
sorry I'll fix it... I've just copy pasted the content of the previous version...